tests/sing_pullfolder_test.cwl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements:
+  DockerRequirement:
+    dockerPull: debian
```

Comment:
I used this because I saw that you had it in another tool you use for testing. It is only to test for .img filename generation. Does this look okay?